lua/lspconfig.lua

```diff
@@ -18,6 +19,60 @@ end
 -- script in scriptnames to be executed is lspconfig.
 function M._root._setup()
   M._root.commands = {
+      LspInfo = {
```

Comment:
Not that it matters or anything but I couldn't help myself, shouldn't this `LspInfo` line gets outdented 2 space to the left? :laughing: